examples/code_generation/config.py

```diff
@@ -0,0 +1,195 @@
+        data_column=["intent", "prompt"],
+        label_column=["test_start", "test", "entry_point"],
+        data_format="odex",
+    ),
+    "humaneval": DatasetConfig(
+        dataset="openai_humaneval",
+        split="test",
+        data_column="prompt",
+        label_column="test",
+        data_format="humaneval",
+    ),
+    "humaneval_lexical": DatasetConfig(
+        dataset="openai_humaneval",
+        split="test",
+        data_column="prompt",
+        label_column="canonical_solution",
+        data_format="humaneval",
+    ),
+}
+
+# The details of each model
+model_configs = {
+    "text-davinci-003": LMConfig(provider="openai", model="text-davinci-003"),
+    "gpt-3.5-turbo": LMConfig(provider="openai_chat", model="gpt-3.5-turbo"),
+    "cohere-command-xlarge": LMConfig(
+        provider="cohere", model="command-xlarge-nightly"
+    ),
+    "gpt2": LMConfig(
+        provider="huggingface",
+        model="gpt2",
+        model_cls=transformers.GPT2LMHeadModel,
+    ),
+    "gpt2-xl": LMConfig(
+        provider="huggingface",
+        model="gpt2-xl",
+        model_cls=transformers.GPT2LMHeadModel,
+    ),
+    "llama-7b": LMConfig(
+        provider="huggingface",
+        model="decapoda-research/llama-7b-hf",
+        tokenizer_cls=transformers.LlamaTokenizer,
+    ),
+    "llama-13b": LMConfig(
+        provider="huggingface",
+        model="decapoda-research/llama-13b-hf",
+        tokenizer_cls=transformers.LlamaTokenizer,
+    ),
+    "alpaca-7b": LMConfig(
+        provider="huggingface",
+        model="chavinlo/alpaca-native",
+    ),
+    "alpaca-13b": LMConfig(
+        provider="huggingface",
+        model="chavinlo/alpaca-13b",
+    ),
+    "vicuna-7b": LMConfig(
+        provider="huggingface",
+        model="eachadea/vicuna-7b-1.1",
+        name_replacements={
+            "system": "ASSISTANT",
+            "assistant": "ASSISTANT",
+            "user": "HUMAN",
+        },
+    ),
+    "codegen-350M-mono": LMConfig(
+        provider="huggingface",
+        model="Salesforce/codegen-350M-mono",
+    ),
+    "codegen-2B-mono": LMConfig(
+        provider="huggingface",
+        model="Salesforce/codegen-2B-mono",
+    ),
+    "codegen-6B-mono": LMConfig(
+        provider="huggingface",
+        model="Salesforce/codegen-6B-mono",
+    )
+}
+
+# The details of the prompts
+prompt_text = {
+    "standard": "Generate Python code solution for the following intent:\n{{source}}\n\n",
+}
+
+# The functions to use to calculate scores for the hyperparameter sweep
+# sweep_distill_functions = [chrf]
+# sweep_metric_function = avg_chrf
+sweep_distill_functions = [execution_accuracy]
+sweep_metric_function = avg_execution_accuracy
+
+# The functions used for Zeno visualization
+zeno_distill_and_metric_functions = [
+    output_length,
+    input_length,
+    label_length,
+    execution_accuracy,
+    # chrf, 
+    # length_ratio,
+    avg_execution_accuracy,
+    # avg_chrf,
+    # avg_length_ratio,
+]
```

Comment:
It's not urgent, but before we merge this we should remove any of these commented-out functions.